src/Symfony/Component/Console/Tests/Tester/CommandTesterTest.php

```diff
@@ -289,4 +291,28 @@ public function testAInvokableExtendedCommand()
 
         $tester->assertCommandIsSuccessful();
     }
+
+    public function testInvokableWithInputAttribute()
+    {
+        $command = new InvokableWithInputTestCommand();
+
+        $tester = new CommandTester($command);
+        $tester->execute([
+            'email' => 'user@example.com',
+            'password' => 'secret',
+            '--admin' => true,
+            '--active' => false,
```

Comment:
will it work with shortcut ? ie if you were to define a shortcut in the option attribute ?

Reply:
yes, it provides all the same features as method parameters, and more: I'm glad I can now use property hooks for implicit normalization